keyboards/aeboards/constellation/rev1/info.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
This makes the `info.json` in the `constellation` dir obsolete.
The cleanest option is probably using `git mv` to move it here so git knows it's the same file.